pkg/controller/migcluster/validation.go

```diff
@@ -129,58 +131,69 @@ func (r ReconcileMigCluster) validateSaSecret(cluster *migapi.MigCluster) error
 
 	// Not needed.
 	if cluster.Spec.IsHostCluster {
+		log.V(4).Info("skipping sa secret validation as this is a host cluster",
+			"name", cluster.Name, "namespace", cluster.Namespace)
 		return nil
 	}
 
 	// NotSet
 	if !migref.RefSet(ref) {
-		cluster.Status.SetCondition(migapi.Condition{
+		cluster.Status.SetAndLogCondition(log, migapi.Condition{
 			Type:     InvalidSaSecretRef,
 			Status:   True,
 			Reason:   NotSet,
 			Category: Critical,
-			Message:  InvalidSaSecretRefMessage,
-		})
+			Message:  "serviceAccountSecretRef not set as a secret for the migcluster",
```

Comment:
Lets be consistent here https://github.com/konveyor/mig-controller/pull/698/files#diff-f817e43dad183fd2d1c8a92bbe814e247f8ea316142836adf4a6301f9729a6e2R190


`"serviceAccountSecretRef name or namespace is empty"`

Reply:
+1, Done